src/Microsoft.Dnx.Host/BootstrapperContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.Versioning;
+
+namespace Microsoft.Dnx.Host
+{
+    public class BootstrapperContext
+    {
+        public string OperatingSystem { get; set; }
+
+        public string OsVersion { get; set; }
+
+        public string Architecture { get; set; }
+
+        public uint PhysicalProcessorCount { get; set; }
```

Comment:
Are you serious? Didn't we literally just cover this in https://github.com/aspnet/dnx/pull/2705#discussion_r39565970 ?